UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/FileShareService.cs

```diff
@@ -574,13 +603,19 @@ public async Task<List<string>> UploadBlockFile(string batchId, CustomFileInfo c
             List<Task> ParallelBlockUploadTasks = new List<Task>();
             long uploadedBytes = 0;
             int blockNum = 0;
-
+            CancellationTokenSource cts = new CancellationTokenSource();
+            CancellationToken cToken = cts.Token;
```

Comment:
Kindly rename variable names to cancellationTokenSource and cancellationToken

Reply:
ok